containers/grizzly2-http/pom.xml

```diff
@@ -41,6 +41,12 @@
             <groupId>org.glassfish.grizzly</groupId>
             <artifactId>grizzly-http-server</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.hamcrest</groupId>
+            <artifactId>hamcrest-junit</artifactId>
+            <version>2.0.0.0</version>
```

Comment:
CQ is for 1.3. Do we need 2.0?

Reply:
Cannot find a version 1.3 for the Maven coordinates `hamcrest-junit` on [Maven Central] (https://search.maven.org/search?q=g:org.hamcrest%20AND%20a:hamcrest-junit&core=gav). It only lists `1.0.0.0-RC1`, `1.0.0.0`, `1.0.0.1` and `2.0.0.0`. So which Maven coordinates shall I use instead?

Reply:
Why you need hamcrest-junit? Only `org.hamcrest.CoreMatchers` and `org.hamcrest.Matchers` are used, neither is sourced in hamcrest-junit, We use hamcrest-library, which should cover the used classes